components/brave_federated/data_stores/data_store_unittest.cc

```diff
@@ -18,15 +20,16 @@
 
 namespace {
 
-struct TestTaskLogTestInfo {
-  int id;
-  int days_from_now;
-  bool label;
-} test_task_log_test_db[] = {
-    {0, 31, true},
-    {0, 15, false},
-    {0, 7, false},
-    {0, 1, true},
+struct TrainingDataTestInfo {
+  int training_instance_id;
+  int feature_name;
+  int feature_type;
+  std::string feature_value;
+} training_data_test_db[] = {
```

Comment:
kTrainingDataTestDb however as this is not a database maybe, kTestTrainingData would read better. your thoughts?

Reply:
agree, went for `kTestTrainingData` 👍